client/blocks/plugins-scheduled-updates/hooks/use-create-monitor.tsx

```diff
@@ -61,11 +61,12 @@ export function useCreateMonitor( siteSlug: SiteSlug ) {
 }
 
 export function useCreateMonitors() {
+	const store = useStore();
 	const { createMonitorSettings } = useBatchCreateMonitorSettingsMutation();
-	const state = useSelector( ( state ) => state );
 
 	const createMonitors = useCallback(
 		( siteSlugs: SiteSlug[] ) => {
+			const state = store.getState();
```

Comment:
`createMonitors` is a callback, read from Redux state at the moment when the callback is called. The `useCreateMonitors` hook itself doesn't ever need to rerender the calling component.